mesh/pmesh.hpp

```diff
@@ -491,6 +491,8 @@ class ParMesh : public Mesh
    void GenerateOffsets(int N, HYPRE_BigInt loc_sizes[],
                         Array<HYPRE_BigInt> *offsets[]) const;
 
+   using Mesh::FaceIsTrueInterior;
```

Comment:
A better approach in my opinion would be to just move the definition of `FaceIsTrueInterior` in `mesh.hpp` into the `public methods`, rather than exposing it here. I would support that change, since `Mesh::FaceIsInterior` is already `public`.

Reply:
I do not know, the thing is there is no good use of `FaceIsTrueInterior` for a non-distributed mesh, because it should coincide with the plain `FaceIsInterior`. It makes only sense for `ParMesh`, but it is not implemented there due to technical reasons, so this approach is maybe better.

Reply:
Because `FaceIsTrueInterior` uses only data members of the base `Mesh` class (in the face informations), its definition should be kept there and moving it would also be a breaking change. I think it is more clean to just make it `public` there and remove this added `using` statement.

Reply:
I agree with you that the implementation should stay there, just I am not sure if we should pollute `Mesh` by something useless there and potentially confusing (the TDOF terminology comes with the distributed meshes). This was probably the reason why this method has been made protected there.

Reply:
I'm not sure I would call moving the method from protected to public polluting, and it adds zero lines of code vs. this change. I'd suggest just going ahead and doing it, or reverting this and opening a separate PR if you think it warrants further discussion.

Reply:
Ok, @vladotomov , any opinion on this?

Reply:
In principle `using` should not be used this way, as it defeats the purpose of the `protected` declaration in the parent; this feels like a hack. But I also agree that `FaceIsTrueInterior` should not be a public method of `Mesh`.

You can create a new public `ParMesh` method that calls `Mesh::FaceIsTrueInterior`, this avoids both of the above.

Reply:
Ok, that sounds to me as a fair compromise 😉